Content.Shared/_NF/GameRule/PointOfInterestPrototype.cs

```diff
@@ -52,29 +52,35 @@ public sealed partial class PointOfInterestPrototype : IPrototype
     public ComponentRegistry AddComponents { get; set; } = new();
 
     /// <summary>
-    ///     If the POI does not belong to a pre-defined group, it will default to the "unique" internal category and will
-    ///     use this float from 0-1 as a raw chance to spawn each round.
+    /// What gamepresets ID this POI is allowed to spawn on.
+    /// </summary>
+    [DataField]
+    public string[] SpawnGamePreset { get; private set; } = [];
+
+    /// <summary>
+    /// If the POI does not belong to a pre-defined group, it will default to the "unique" internal category and will
+    /// use this float from 0-1 as a raw chance to spawn each round.
     /// </summary>
     [DataField]
     public float SpawnChance { get; private set; } = 1;
 
     /// <summary>
-    ///     The group that this POI belongs to. Currently, the default groups are:
-    ///     "CargoDepot"
-    ///     "MarketStation"
-    ///     "Required"
-    ///     "Optional"
-    ///     Each POI labeled in the Required group will be spawned in every round.
-    ///     Apart from that, each of thesehave corresponding CVARS by default, that set an optional # of this group to spawn.
-    ///     Traditionally, it is 2 cargo depots, 1 trade station, and 8 optional POIs.
-    ///     Dynamically added groups will default to 1 option chosen in that group, using the SpawnChance as a weighted chance
-    ///     for the entire group to spawn on a per-POI basis.
+    /// The group that this POI belongs to. Currently, the default groups are:
+    /// "CargoDepot"
+    /// "MarketStation"
+    /// "Required"
+    /// "Optional"
+    /// Each POI labeled in the Required group will be spawned in every round.
+    /// Apart from that, each of thesehave corresponding CVARS by default, that set an optional # of this group to spawn.
+    /// Traditionally, it is 2 cargo depots, 1 trade station, and 8 optional POIs.
+    /// Dynamically added groups will default to 1 option chosen in that group, using the SpawnChance as a weighted chance
+    /// for the entire group to spawn on a per-POI basis.
```

Comment:
this reformatting here bulldozes IDE presenting meta data, and goes against the conventions throughout the rest of the code base

Reply:
Weird this is how its done on some other files, but I think no one fixed it
Ill fix it

Reply:
yea its following convention because tags in comments like `summary` get handled by IDEs to create context-sensitive rich UI and generally helps broader maintainability. Most important for public APIs, so you should probably write new ones similar to others you see in the new POI system since you are now intending that system to be publicly accessable/shared